ansible/roles/post-recheck-comment/tasks/main.yaml

```diff
@@ -0,0 +1,64 @@
+---
+
+- name: Store Verified-1 regex pattern
+  set_fact:
+    failure_pattern: '^Patch Set {{ patchset }}: Verified-1'
+
+- name: Wait for Verified-1 comment from {{ ci_user }}
+  register: comments
+  raw: gerrit query change:{{ change_id }} --comments --format JSON
+  retries: 6
+  delay: 10
+  until: >
+    comments.rc == 0 and
+    (comments.stdout_lines[0] | from_json).comments
+    | selectattr('reviewer.username', 'equalto', ci_user)
+    | map(attribute='message')
+    | list
+    | last
+    | regex_search(failure_pattern)
+  tags:
+    # This would fire [ANSIBLE0012] Commands should not change things if nothing needs doing,
+    # but it's ok as this command is side-effect free.
+    - skip_ansible_lint
+
+- name: Count failures
+  set_fact:
+    n_failures: >
+      {{
+      (comments.stdout_lines[0] | from_json).comments
+      | selectattr('reviewer.username', 'equalto', ci_user)
+      | map(attribute='message')
+      | map('regex_search', failure_pattern)
+      | select
+      | list
+      | count
```

Comment:
Since `n_failures` is used as an int later, couldn't we use `int` filter here?

Then `int` filter from next tasks could be removed.

Reply:
This applies to `max_rechecks` as well.

Reply:
Funny fact – no. I've tested it and unfortunately somehow it gets stringified somewhere and doesn't work.